relay_sidecar/cli.py

```
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+from .client import RelaySidecar, SidecarConfig
+
+
+async def pull_once(config_path: Path) -> None:
+    config = SidecarConfig.from_toml(
+        data=__load_toml(config_path)
+    )
+    sidecar = RelaySidecar(config)
+    # Run a single polling cycle across configured mailboxes, then exit
+    await sidecar._poll_cycle()  # noqa: SLF001 (intentional internal use)
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)